install/tools/ipa-pwdreader.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+HTTP_PASSWD_LOC="/var/lib/ipa/certs/httpd_passwd.txt"
+
+if [ "$(basename $0)" == "ipa-httppswd.sh" ] && \
+    [ -f "$HTTP_PASSWD_LOC" ]; then
+    cat "$HTTP_PASSWD_LOC"
```

Comment:
[SSLPassPhraseDialog](https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslpassphrasedialog) has context ``server_config``. It means that mod_ssl has only one option for the entire process. Although it is not supported, users tend to deploy additional services and virtual hosts on the same machine. Therefore this script must only return the httpd password for IPA's vhost.

Reply:
It does.

Reply:
How is this suppose to work? There can only be one command for all vhosts. You have to check argv 1 and 2 to detect the correct vhost.

Reply:
Ah, the response only applies for the "Therefore this script must only return the httpd password for IPA's vhost."
I did not consider support for more because we did not support anything else running alongside IPA in the HTTPD service.